demos/http/http_demo_plaintext/demo_config.h

```diff
@@ -43,4 +43,65 @@
 
 /************ End of logging configuration ****************/
 
+/**
+ * @brief HTTP server host name.
```

Comment:
Great change!

Reply:
Yah the demo was looking a little too long from all these macros lol